src/machine/behavior/is/mod.rs

```diff
@@ -19,15 +19,263 @@ pub fn get(_opcode: u8) -> Semantic {
 }
 
 type Semantic = fn(&mut State, u8, u8, u8);
-/*
-pub const SEMANTICS: [Semantic; 256] = [
-                        // symbol   hex  dec
-
-    interrupt::trap,    // TRAP     #00    0
-    float::fcmp,        // FCMP     #01    1
-
-    /* TODO */          // PUSHGOI  #BF  191
 
-    other::swym         // SWYM     #FF  255
+pub const SEMANTICS: [Semantic; 256] = [
+    interrupt::trap,    
+    float::fcmp,      
+    float::fun,       
+    float::feql,      
+    float::fadd,      
+    float::fix,       
+    float::fsub,      
+    float::fixu,        
+    float::flot,
+    float::floti,
+    float::flotu,
+    float::flotui,
+    float::sflot,
+    float::sfloti,
+    float::sflotu,
+    float::sflotui,
+    float::fmul,
+    float::fcmpe,
+    float::fune,
+    float::feqle,
+    float::fdiv,
+    float::fsqrt,
+    float::frem,
+    float::fint,
+    arithmetic::mul,
+    arithmetic::muli,
+    arithmetic::mulu,
+    arithmetic::mului,
+    arithmetic::div,
+    arithmetic::divi,
+    arithmetic::divu,
+    arithmetic::divui,
+    arithmetic::add,
+    arithmetic::addi,
+    arithmetic::addu,
+    arithmetic::addui,
+    arithmetic::sub,
+    arithmetic::subi,
+    arithmetic::subu,
+    arithmetic::subui,
+    arithmetic::addu2,
+    arithmetic::addu2i,
+    arithmetic::addu4,
+    arithmetic::addu4i,
+    arithmetic::addu8,
+    arithmetic::addu8i,
+    arithmetic::addu16,
```

Comment:
As I see, you really changed the order of addu and 16 to fit into Rust's name scheme. I don't really like that but as long as I have no better option I support to go ahead with that.

Does someone else have an idea?